### PR TITLE
Standardize projects

### PR DESCRIPTION
**Pull request by Tyrrrz**
Rename projects to match assembly names, clean up. Does not involve any breaking changes.

### CODE REVIEW

**Comment by Tyrrrz** [comment]
A bit confused in regards to `dotnet format` errors, can't reproduce it locally. Can anyone help out?

File: .editorconfig [hunk 1]
```diff
@@ -18,7 +18,7 @@ indent_size = 4
 tab_width = 4
 
 # New line preferences
-end_of_line = crlf
+end_of_line = lf
```
Comment (Tyrrrz) [inline .editorconfig:21]: Switched this to ensure consistent formatting between Windows/non-Windows OS

Comment (jrmccannon) [reply .editorconfig:21]: If we didn't want to update this, you can update your git config and rider to ensure it always commits properly. No preference personally. LF would make things easier for new contributors.

**Review by jrmccannon — APPROVED** [review]

**Review by abergs — APPROVED** [review]
Looks good to me. Wondering if @justindbaur has any comments? Note that there's a small conflict.

**Comment by justindbaur** [comment]
So this was born out of a practice I inherited from `bitwarden/server`. But I think there is sufficient reason that the pros I get from it in there aren't needed here. 

There I like to just be able to do `Ctrl+P` type in `Admin` and see the `.csproj` file right there near the top and not have to constantly type through the prefix to have confidence it will be there at the type. But in this repo, there is a lot less going on AND it is more common to have things that are nuget packages, have the exact name in the `csproj` as the package id so it could be easier to find. So for that I think we should instead make the names `Passwordless` and `Passwordless.Tests`.

**Comment by Tyrrrz** [comment]
> ...have the exact name in the `csproj` as the package id so it could be easier to find. So for that I think we should instead make the names `Passwordless` and `Passwordless.Tests`.

One thing to point out is that the package id and the assembly/namespace names are still going to be mismatched here because the package id is `Passwordless` while the latter two are `Passwordless.Net`. However, that reduces mismatches from 2 to 1 at least 🙂 The other issue is not possible to solve without a breaking change.

**Comment by abergs** [comment]
@Tyrrrz @justindbaur Breaking changes have way less impact now than in 6 months.

**Comment by justindbaur** [comment]
@Tyrrrz Yeah I think that was a miscommunication between @abergs and I, I sorta expected the package id to be `Passwordless.Net` (following `[Company].[Product]` and being a little cheeky with the pronunciation). I think we should for sure make the package file name equal to the package id and I'm fine with changing the namespace but I leave that to you all.

**Comment by Tyrrrz** [comment]
@abergs @justindbaur Makes sense. Should I change the namespace to just `Passwordless` then to match the package ID?

image

**Pull request merged**